source/files/stylesheets/adjustments.css

```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
   transition: color 0.2s linear;
 }
 
-.layout-footer .footer-row-1 {
-  margin: 0 auto;
-/*
-  max-width: 67em;
- */
-  padding: 0 10px;
-  min-height: 260px;
-}
-
-.layout-footer .footer-row-1:before,
-.layout-footer .footer-row-1:after {
-  content: "";
-/*
```

Comment:
This is an IE hack; without it, the footer will bleed into the content on IE 11.